README.md

```diff
@@ -4,43 +4,45 @@ A GitHub App that helps maintain a large number of repositories in a GitHub orga
 
 [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
 
-<img width="30%" align="left" src="https://user-images.githubusercontent.com/224547/46536555-c29d1180-c8a6-11e8-92c2-f3141da0d6da.png" />
+<img width="10%" align="left" src="https://user-images.githubusercontent.com/224547/46536555-c29d1180-c8a6-11e8-92c2-f3141da0d6da.png" />
 
-<img width="50%" align="" src="https://user-images.githubusercontent.com/224547/46534085-e65c5980-c89e-11e8-90b7-06e060217de1.png" />
+<img width="30%" align="" src="https://user-images.githubusercontent.com/224547/46534085-e65c5980-c89e-11e8-90b7-06e060217de1.png" />
 
-## Setup
+## Development
+
+### Getting Started
+
+1. Build the application
 
 ```sh
-# Install dependencies
+# Install the correct version of Node.js (defined in .nvmrc)
+nvm use
```

Comment:
Oh no! What if I'm not using `nvm`?! :scream:

Reply:
As in I should list other tools? Or?

Reply:
I was thinking it should say something along the lines of "if you use nvm", as it currently implies that you _need_ to be using `npm`.